packages/protoc-gen-es/src/typescript.ts

```diff
@@ -101,7 +102,7 @@ function generateMessage(schema: Schema, f: GeneratedFile, message: DescMessage)
   f.print("  }");
   f.print();
   generateWktMethods(schema, f, message);
-  f.print("  static readonly runtime = ", protoN, ";");
+  f.print("  static readonly runtime: ", ProtoRuntime, " = ", protoN, ";");
```

Comment:
I noticed that declaration.ts generates `runtime: typeof proto3 = proto3` here. 

And this actually works for typescript.ts as well. This passes the tests: 

```suggestion
  f.print("  static readonly runtime: typeof ", protoN ," = ", protoN, ";");
```

I think this is preferable over `ProtoRuntime`. We will add another runtime soon, and `ProtoRuntime` would no longer fit, and we would have to export even more types.

Can you make this change? That means we don't actually have to export `ProtoRuntime` anymore, and we don't have to loop it through codegen-info.ts and runtime-imports.ts, which is great.

Reply:
Ah yeah, that's much better.